### PR TITLE
add tests for ensuring ownership for opening live archive previously …

### PR DESCRIPTION
**Pull request by clkao**
…created

### CODE REVIEW

File: test/owner.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    t.ok(archive.owner)
+    t.is(secretKey.length, 128)
+  })
+})
+
+test('owner', function (t) {
+  t.plan(2)
+  var drive = hyperdrive(db)
+  var archive = drive.createArchive(key, { live: true })
+  console.log(archive.metadata.secretKey)
```
Comment (ralphtheninja) [inline test/owner.js]: Remove this?

Comment (clkao) [inline test/owner.js]: nice catch!

**Comment by mafintosh** [comment]
awesome

**Comment by karissa** [comment]
woot

**Pull request merged**